python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -1911,10 +1911,6 @@ def doit(f):
             logger.debug("Remove Err: %s" % str(err))
             pass
 
-    def fail(self, force=False):
```

Comment:
Thanks @rob-c I'd missed this. I guess this is another reason to put together a patch to move to using the `@export` decorator I developed.